aviatrix/resource_aviatrix_vpc.go

```diff
@@ -217,6 +225,11 @@ func resourceAviatrixVpcCreate(d *schema.ResourceData, meta interface{}) error {
 			return fmt.Errorf("advanced option('subnet_size' and 'num_of_subnet_pairs') is only supported for AWS and Azure provider")
 		}
 	}
+	if vpc.EnablePrivateOobSubnet {
+		if vpc.CloudType != goaviatrix.AWS && vpc.CloudType != goaviatrix.AWSGOV {
```

Comment:
Nit: This could all just be in a single if statement, but it's a style choice and we haven't defined what we want to do in the style guide about this yet so no big deal.

Reply:
This way is easier to read and seems cleaner. We will define a style in future.